v2/apiserver/internal/api/containers.go

```diff
@@ -23,3 +23,39 @@ type ContainerSpec struct {
 	// to be set within the OCI container.
 	Environment map[string]string `json:"environment,omitempty" bson:"environment,omitempty"` // nolint: lll
 }
+
+func (cs ContainerSpec) EqualTo(cs2 ContainerSpec) bool {
+	// Compare Command slices
+	if len(cs.Command) != len(cs2.Command) {
+		return false
+	}
+	for i, command := range cs.Command {
+		if command != cs2.Command[i] {
+			return false
+		}
+	}
```

Comment:
Can these really not be compared without iterating? May be the case. I cannot recall.

Reply:
Yes, I believe these slices must be iterated for comparison.